helm-charts/konk/templates/etcd.yaml

```diff
@@ -0,0 +1,27 @@
+{{- with .Values.etcd }}
+{{- if .operator }}
+apiVersion: konk.infoblox.com/v1alpha1
+kind: Etcd
+metadata:
+  name: {{ $.Release.Name }}-etcd
+  namespace: {{ $.Release.Namespace }}
+  labels:
+    {{- include "konk.labels" $ | nindent 4 }}
+spec:
+  auth:
+    rbac:
+      enabled: false
+    client:
+      secureTransport: true
+      existingSecret: {{ include "konk.fullname" $ }}-etcd-cert
+      certFilename: server.crt
+      certKeyFilename: server.key
+  # these probes don't work with self-signed certs because probe.sh doesn't set the cacert path in the $AUTH_OPTIONS
```

Comment:
Probably worth mentioning in the comments that readiness probes don't work well with Etcd. For cluster initialization, the pods need to accept traffic prior to being ready to do initial gossip. They can talk to each other on pod IPs, but that's a bit fragile.
https://github.com/improbable-eng/etcd-cluster-operator/blob/master/docs/operations.md#frequently-asked-questions